### ISSUE TITLE
File service + whiteboard container service restart if no server

### ISSUE THREAD

**Issue by techsmyth** [alkem-io/file-service-archive#9]
## Description

As a developer I want robust local services

## Acceptance criteria

- [ ] The services should not restart if the server is not available, but look to reconnect periodically

## Additional Context
The containers are failing with the following message:
<img width="1387" src="https://api.zenhub.com/attachedFiles/eyJfcmFpbHMiOnsibWVzc2FnZSI6IkJBaHBBNTlRQlE9PSIsImV4cCI6bnVsbCwicHVyIjoiYmxvYl9pZCJ9fQ==--d8bcfff76b552ac8a8c491839d226ec841941e7c/image.png" alt="image.png" />

**Comment by hero101** [comment]
@techsmyth Which server is not available? This is a Winston error and not related to any dependant service.
The container will restart and "fix itself" in the proccess. Only local docker containers are affected.

**Comment by techsmyth** [comment]
@hero101 yes indeed it is the local dev setup that is having this issue, Valentin + myself were busy for a while looking at why the containers were restarting in a loop locally, and then I noticed when I started the server locally they were happy

**Comment by hero101** [comment]
> @hero101 yes indeed it is the local dev setup that is having this issue, Valentin + myself were busy for a while looking at why the containers were restarting in a loop locally, and then I noticed when I started the server locally they were happy

It has nothing to do with the server, unfortunatelly. I was not able to reproduce this behaviour, and the services do not directly connect to the server, but rather to rabbitMQ, which is a retriable and the connection is ensured.
